src/CarterRoute.cs

```diff
@@ -0,0 +1,24 @@
+namespace Carter
+{
+    using System;
+    using Microsoft.AspNetCore.Http;
+
+    public class CarterRoute
```

Comment:
The introduction of this seems like it might be useful for implementing the `.Pipe(...)` functionality mentioned in #112 👍

Reply:
I actually already have a version of the pipe stuff ready but I'm saving it for the next PR lol